pkg/events/parse/params_test.go

```diff
@@ -0,0 +1,115 @@
+package parse
+
+import (
+	"testing"
+
+	"github.com/aquasecurity/tracee/types/trace"
+)
+
+func TestArgInt32Val(t *testing.T) {
+	e := trace.Event{Args: []trace.Argument{
```

Comment:
@ShubhamPalriwala can we use the table test here and check for the 3 cases of each method?

- valid value (we should check the return value)
- invalid argument type
- argument not found

Reply:
Oh okay! Sorry for not following the format. On it!